schemas/business/en/epe_0003.json

```diff
@@ -26,27 +27,20 @@
           "title": "Introduction",
           "blocks": [
             {
-              "id": "introduction-block",
+              "id": "introduction",
               "type": "Introduction",
               "primary_content": [
                 {
                   "id": "primary",
                   "title": {
-                    "text": "You are completing this for {ru_name} {conditional_trad_as}",
+                    "text": "You are completing this for {trad_as} ({ru_name})",
                     "placeholders": [
```

Comment:
The previous `conditional_trad_as` block will be required here, to handle conditions when `trad_as` is empty.
```
      "placeholder": "conditional_trad_as",
                        "transforms": [
                          {
                            "arguments": {
                              "trad_as": {
                                "source": "metadata",
                                "identifier": "trad_as"
                              }
                            },
                            "transform": "conditional_trad_as"
                          }
                        ]
```